src/org/loklak/data/DAO.java

```diff
@@ -978,6 +987,25 @@ public SearchLocalMessages(final String q, final Timeline.Order order_field, fin
             }
             this.aggregations = query.aggregations;
         }
+
+        public SearchLocalMessages (
+                final String q,
+                final Timeline.Order order_field,
+                final int timezoneOffset,
+                final int resultCount,
+                final int aggregationLimit,
+                final String... aggregationFields
+        ) {
+            this(
+                q,
+                order_field,
+                timezoneOffset,
+                resultCount,
+                aggregationLimit,
+                new ArrayList<>(),
+                aggregationFields
+            );
+        }
```

Comment:
Can you accommodate theses variables in a single line? The allowable line length is enough to support this. Also, it will be consistent with other parts of the project. There are many other such changes in this PR.